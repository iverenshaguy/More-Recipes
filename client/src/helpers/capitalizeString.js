/**
 * Capitalizes the first letter in a string
 * @function capitalizeString
 * @param {string} str
 * @returns {string} Returns a string with the first letter capitalized
 */
function capitalizeString(str) {
  return `${str.charAt(0).toUpperCase()}${str.substr(1)}`;
}

export default capitalizeString;
