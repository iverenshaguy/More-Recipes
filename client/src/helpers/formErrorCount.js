/**
 * Returns number of errors in form
 * @function formErrorCount
 * @param {object} formErrors
 * @returns {number} Number of errors
 */
const formErrorCount = formErrors => Object.values(formErrors).filter((value) => {
  let isError;
  if (Array.isArray(value)) {
    const filterError = (Object.values(value).filter(val => val !== null)).length;
    isError = filterError ? 'Error' : null;
  } else {
    isError = value;
  }
  return isError !== null;
}).length;

export default formErrorCount;
