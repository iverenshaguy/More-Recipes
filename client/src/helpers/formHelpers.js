import { login, signup, clearAuthError } from '../actions/auth';

const clearFormError = {
  login: clearAuthError(),
  signup: clearAuthError()
};

const formFields = {
  login: ['email', 'password'],
  signup: [
    'firstname',
    'lastname',
    'username',
    'email',
    'password',
    'passwordConfirm',
    'occupation',
    'aboutMe'
  ]
};

const formSubmitMapper = {
  login,
  signup
};

export default {
  formFields,
  clearFormError,
  formSubmitMapper
};
