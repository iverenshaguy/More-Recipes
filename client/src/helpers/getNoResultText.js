const getNoResultText = (title) => {
  switch (title) {
    case 'TOP RECIPES':
      return 'There are no top recipes at the moment';
    case 'SEARCH RESULTS':
      return 'Your search returned no results';
    case 'REVIEWS RESULT':
      return 'There are no reviews for this recipe';
    case 'MY RECIPES':
      return 'You have no recipe yet, add a recipe to see your recipes';
    default:
      return 'There are no recipes at the moment';
  }
};

export default getNoResultText;
