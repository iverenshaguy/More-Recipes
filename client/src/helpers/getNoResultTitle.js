const getNoResultTitle = (title) => {
  switch (title) {
    case 'TOP RECIPES':
      return 'There are no top recipes at the moment';
    case 'SEARCH RESULTS':
      return 'Your search returned no results';
    default:
      return 'There are no recipes at the moment';
  }
};

export default getNoResultTitle;
