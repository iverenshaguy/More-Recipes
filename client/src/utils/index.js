import errorHandler from './errorHandler';
import arrayToObject from './arrayToObject';
import fileEventAdapter from './fileEventAdapter';

export default {
  errorHandler,
  arrayToObject,
  fileEventAdapter,
};
