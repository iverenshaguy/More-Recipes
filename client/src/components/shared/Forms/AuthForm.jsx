import React, { Fragment } from 'react';
import { RenderInput } from '../FormComponents';
import { formPropTypes } from '../../../helpers/proptypes';

const placeholder = '\u2022\u2022\u2022\u2022\u2022\u2022\u2022\u2022\u2022\u2022';

/**
 * @exports
 * @function AuthForm
 * @param {object} props - props
 * @returns {component} AuthForm
 */
const AuthForm = ({ type, state, handlers }) => {
  AuthForm.propTypes = {
    ...formPropTypes(type)
  };

  return (
    <Fragment>
      {type === 'signup' && <RenderInput
        type="text"
        name="firstname"
        label="First Name"
        required
        id="firstname"
        value={state.values.firstname}
        placeholder="Jane"
        handleChange={handlers.handleChange}
        handleBlur={handlers.handleBlur}
        handleFocus={handlers.handleFocus}
        meta={{
          touched: state.touched.firstname,
          error: state.error.firstname
        }}
      />}
      {type === 'signup' && <RenderInput
        type="text"
        name="lastname"
        id="lastname"
        label="Last Name"
        required={false}
        value={state.values.lastname}
        placeholder="Smith"
        handleChange={handlers.handleChange}
        handleBlur={handlers.handleBlur}
        handleFocus={handlers.handleFocus}
        meta={{
          touched: state.touched.lastname,
          error: state.error.lastname,
        }}
      />}
      <RenderInput
        type="email"
        name="email"
        label="Email Address"
        id="email"
        required={type !== 'login'}
        value={state.values.email}
        placeholder="janesmith@me.com"
        handleChange={handlers.handleChange}
        handleBlur={handlers.handleBlur}
        handleFocus={handlers.handleFocus}
        meta={{
          touched: state.touched.email,
          error: state.error.email,
          asyncValidating: state.asyncValidating
        }}
      />
      {type === 'signup' && <RenderInput
        type="text"
        name="username"
        id="username"
        label="Username"
        required
        value={state.values.username}
        placeholder="janesmith"
        handleChange={handlers.handleChange}
        handleBlur={handlers.handleBlur}
        handleFocus={handlers.handleFocus}
        meta={{
          touched: state.touched.username,
          error: state.error.username,
          asyncValidating: state.asyncValidating
        }}
      />}
      <RenderInput
        type="password"
        name="password"
        className="mb-4"
        label="Password"
        id="password"
        required={type !== 'login'}
        value={state.values.password}
        placeholder={placeholder}
        handleChange={handlers.handleChange}
        handleBlur={handlers.handleBlur}
        handleFocus={handlers.handleFocus}
        meta={{
          touched: state.touched.password,
          error: state.error.password,
        }}
      />
      {type === 'signup' && <RenderInput
        type="password"
        name="passwordConfirm"
        id="passwordconfirm"
        label="Password Confirmation"
        required
        value={state.values.passwordConfirm}
        handleChange={handlers.handleChange}
        handleBlur={handlers.handleBlur}
        handleFocus={handlers.handleFocus}
        meta={{
          touched: state.touched.passwordConfirm,
          error: state.error.passwordConfirm
        }}
      />}
      {type === 'signup' && <RenderInput
        type="textarea"
        name="aboutMe"
        id="user-info"
        label="Tell Us Something About Yourself"
        required={false}
        value={state.values.aboutMe}
        placeholder="I love to cook"
        handleChange={handlers.handleChange}
        handleBlur={handlers.handleBlur}
        handleFocus={handlers.handleFocus}
        meta={{
          touched: state.touched.aboutMe,
          error: state.error.aboutMe
        }}
      />}
      {type === 'signup' && <RenderInput
        type="text"
        name="occupation"
        id="occupation"
        label="Occupation"
        required={false}
        value={state.values.occupation}
        placeholder="Chef"
        handleChange={handlers.handleChange}
        handleBlur={handlers.handleBlur}
        handleFocus={handlers.handleFocus}
        meta={{
          touched: state.touched.occupation,
          error: state.error.occupation
        }}
      />}
    </Fragment>
  );
};

export default AuthForm;
