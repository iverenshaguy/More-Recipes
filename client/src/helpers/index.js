import formHelpers from './formHelpers';
import validations from './validations';
import getNoResultTitle from './getNoResultTitle';

export default {
  formHelpers,
  validations,
  getNoResultTitle
};
