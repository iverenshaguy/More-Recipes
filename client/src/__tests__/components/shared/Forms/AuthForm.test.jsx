import React from 'react';
import { MemoryRouter } from 'react-router-dom';
import AuthForm from '../../../../components/shared/Forms/AuthForm';
import { arrayToObject } from '../../../../utils';
import { formHelpers } from '../../../../helpers';

const dispatchMock = jest.fn();
const { formFields } = formHelpers;
const setup = (type) => {
  const props = {
    type,
    state: {
      values: arrayToObject(formFields[type], ''),
      touched: arrayToObject(formFields[type], false),
      error: arrayToObject(formFields[type], null),
      pristine: true,
      formValid: false,
      asyncValidating: false
    },
    handlers: {
      handleBlur: jest.fn(),
      handleChange: jest.fn(),
      handleFocus: jest.fn(),
      handleSubmit: jest.fn()
    }
  };

  const mountRoot = mount( //eslint-disable-line
    <MemoryRouter>
      <AuthForm {...props} dispatch={dispatchMock} />
    </MemoryRouter>);

  return { props, mountRoot };
};

describe('AuthForm', () => {
  afterAll(() => {
    jest.clearAllMocks();
  });

  it('renders correctly when type is login', () => {
    const { props } = setup('login');

    const shallowComponent = mount(<AuthForm {...props} />);

    expect(toJson(shallowComponent)).toMatchSnapshot();
  });

  it('renders correctly when type is signup', () => {
    const { props } = setup('signup');
    const shallowComponent = mount(<AuthForm {...props} />);

    expect(toJson(shallowComponent)).toMatchSnapshot();
  });
});
