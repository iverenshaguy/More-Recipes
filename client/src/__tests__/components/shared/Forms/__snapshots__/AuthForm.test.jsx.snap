// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AuthForm renders correctly when type is login 1`] = `
<AuthForm
  handlers={
    Object {
      "handleBlur": [Function],
      "handleChange": [Function],
      "handleFocus": [Function],
      "handleSubmit": [Function],
    }
  }
  state={
    Object {
      "asyncValidating": false,
      "error": Object {
        "email": null,
        "password": null,
      },
      "formValid": false,
      "pristine": true,
      "touched": Object {
        "email": false,
        "password": false,
      },
      "values": Object {
        "email": "",
        "password": "",
      },
    }
  }
  type="login"
>
  <RenderInput
    handleBlur={[Function]}
    handleChange={[Function]}
    handleFocus={[Function]}
    id="email"
    label="Email Address"
    meta={
      Object {
        "asyncValidating": false,
        "error": null,
        "touched": false,
      }
    }
    name="email"
    placeholder="janesmith@me.com"
    required={false}
    type="email"
    value=""
  >
    <FormGroup
      tag="div"
    >
      <div
        className="form-group"
      >
        <Label
          className="col-form-label"
          for="email"
          tag="label"
          widths={
            Array [
              "xs",
              "sm",
              "md",
              "lg",
              "xl",
            ]
          }
        >
          <label
            className="col-form-label  form-control-label"
            htmlFor="email"
          >
            Email Address
          </label>
        </Label>
        <Input
          className=""
          id="email"
          name="email"
          onBlur={[Function]}
          onChange={[Function]}
          onFocus={[Function]}
          placeholder="janesmith@me.com"
          tag="p"
          type="email"
          value=""
        >
          <input
            className="form-control"
            id="email"
            name="email"
            onBlur={[Function]}
            onChange={[Function]}
            onFocus={[Function]}
            placeholder="janesmith@me.com"
            type="email"
            value=""
          />
        </Input>
      </div>
    </FormGroup>
  </RenderInput>
  <RenderInput
    className="mb-4"
    handleBlur={[Function]}
    handleChange={[Function]}
    handleFocus={[Function]}
    id="password"
    label="Password"
    meta={
      Object {
        "error": null,
        "touched": false,
      }
    }
    name="password"
    placeholder="••••••••••"
    required={false}
    type="password"
    value=""
  >
    <FormGroup
      tag="div"
    >
      <div
        className="form-group"
      >
        <Label
          className="col-form-label"
          for="password"
          tag="label"
          widths={
            Array [
              "xs",
              "sm",
              "md",
              "lg",
              "xl",
            ]
          }
        >
          <label
            className="col-form-label  form-control-label"
            htmlFor="password"
          >
            Password
          </label>
        </Label>
        <Input
          className=""
          id="password"
          name="password"
          onBlur={[Function]}
          onChange={[Function]}
          onFocus={[Function]}
          placeholder="••••••••••"
          tag="p"
          type="password"
          value=""
        >
          <input
            className="form-control"
            id="password"
            name="password"
            onBlur={[Function]}
            onChange={[Function]}
            onFocus={[Function]}
            placeholder="••••••••••"
            type="password"
            value=""
          />
        </Input>
      </div>
    </FormGroup>
  </RenderInput>
</AuthForm>
`;

exports[`AuthForm renders correctly when type is signup 1`] = `
<AuthForm
  handlers={
    Object {
      "handleBlur": [Function],
      "handleChange": [Function],
      "handleFocus": [Function],
      "handleSubmit": [Function],
    }
  }
  state={
    Object {
      "asyncValidating": false,
      "error": Object {
        "aboutMe": null,
        "email": null,
        "firstname": null,
        "lastname": null,
        "occupation": null,
        "password": null,
        "passwordConfirm": null,
        "username": null,
      },
      "formValid": false,
      "pristine": true,
      "touched": Object {
        "aboutMe": false,
        "email": false,
        "firstname": false,
        "lastname": false,
        "occupation": false,
        "password": false,
        "passwordConfirm": false,
        "username": false,
      },
      "values": Object {
        "aboutMe": "",
        "email": "",
        "firstname": "",
        "lastname": "",
        "occupation": "",
        "password": "",
        "passwordConfirm": "",
        "username": "",
      },
    }
  }
  type="signup"
>
  <RenderInput
    handleBlur={[Function]}
    handleChange={[Function]}
    handleFocus={[Function]}
    id="firstname"
    label="First Name"
    meta={
      Object {
        "error": null,
        "touched": false,
      }
    }
    name="firstname"
    placeholder="Jane"
    required={true}
    type="text"
    value=""
  >
    <FormGroup
      tag="div"
    >
      <div
        className="form-group"
      >
        <Label
          className="col-form-label"
          for="firstname"
          tag="label"
          widths={
            Array [
              "xs",
              "sm",
              "md",
              "lg",
              "xl",
            ]
          }
        >
          <label
            className="col-form-label  form-control-label"
            htmlFor="firstname"
          >
            First Name
            <span
              className="text-danger"
            >
              *
            </span>
          </label>
        </Label>
        <Input
          className=""
          id="firstname"
          name="firstname"
          onBlur={[Function]}
          onChange={[Function]}
          onFocus={[Function]}
          placeholder="Jane"
          tag="p"
          type="text"
          value=""
        >
          <input
            className="form-control"
            id="firstname"
            name="firstname"
            onBlur={[Function]}
            onChange={[Function]}
            onFocus={[Function]}
            placeholder="Jane"
            type="text"
            value=""
          />
        </Input>
      </div>
    </FormGroup>
  </RenderInput>
  <RenderInput
    handleBlur={[Function]}
    handleChange={[Function]}
    handleFocus={[Function]}
    id="lastname"
    label="Last Name"
    meta={
      Object {
        "error": null,
        "touched": false,
      }
    }
    name="lastname"
    placeholder="Smith"
    required={false}
    type="text"
    value=""
  >
    <FormGroup
      tag="div"
    >
      <div
        className="form-group"
      >
        <Label
          className="col-form-label"
          for="lastname"
          tag="label"
          widths={
            Array [
              "xs",
              "sm",
              "md",
              "lg",
              "xl",
            ]
          }
        >
          <label
            className="col-form-label  form-control-label"
            htmlFor="lastname"
          >
            Last Name
          </label>
        </Label>
        <Input
          className=""
          id="lastname"
          name="lastname"
          onBlur={[Function]}
          onChange={[Function]}
          onFocus={[Function]}
          placeholder="Smith"
          tag="p"
          type="text"
          value=""
        >
          <input
            className="form-control"
            id="lastname"
            name="lastname"
            onBlur={[Function]}
            onChange={[Function]}
            onFocus={[Function]}
            placeholder="Smith"
            type="text"
            value=""
          />
        </Input>
      </div>
    </FormGroup>
  </RenderInput>
  <RenderInput
    handleBlur={[Function]}
    handleChange={[Function]}
    handleFocus={[Function]}
    id="email"
    label="Email Address"
    meta={
      Object {
        "asyncValidating": false,
        "error": null,
        "touched": false,
      }
    }
    name="email"
    placeholder="janesmith@me.com"
    required={true}
    type="email"
    value=""
  >
    <FormGroup
      tag="div"
    >
      <div
        className="form-group"
      >
        <Label
          className="col-form-label"
          for="email"
          tag="label"
          widths={
            Array [
              "xs",
              "sm",
              "md",
              "lg",
              "xl",
            ]
          }
        >
          <label
            className="col-form-label  form-control-label"
            htmlFor="email"
          >
            Email Address
            <span
              className="text-danger"
            >
              *
            </span>
          </label>
        </Label>
        <Input
          className=""
          id="email"
          name="email"
          onBlur={[Function]}
          onChange={[Function]}
          onFocus={[Function]}
          placeholder="janesmith@me.com"
          tag="p"
          type="email"
          value=""
        >
          <input
            className="form-control"
            id="email"
            name="email"
            onBlur={[Function]}
            onChange={[Function]}
            onFocus={[Function]}
            placeholder="janesmith@me.com"
            type="email"
            value=""
          />
        </Input>
      </div>
    </FormGroup>
  </RenderInput>
  <RenderInput
    handleBlur={[Function]}
    handleChange={[Function]}
    handleFocus={[Function]}
    id="username"
    label="Username"
    meta={
      Object {
        "asyncValidating": false,
        "error": null,
        "touched": false,
      }
    }
    name="username"
    placeholder="janesmith"
    required={true}
    type="text"
    value=""
  >
    <FormGroup
      tag="div"
    >
      <div
        className="form-group"
      >
        <Label
          className="col-form-label"
          for="username"
          tag="label"
          widths={
            Array [
              "xs",
              "sm",
              "md",
              "lg",
              "xl",
            ]
          }
        >
          <label
            className="col-form-label  form-control-label"
            htmlFor="username"
          >
            Username
            <span
              className="text-danger"
            >
              *
            </span>
          </label>
        </Label>
        <Input
          className=""
          id="username"
          name="username"
          onBlur={[Function]}
          onChange={[Function]}
          onFocus={[Function]}
          placeholder="janesmith"
          tag="p"
          type="text"
          value=""
        >
          <input
            className="form-control"
            id="username"
            name="username"
            onBlur={[Function]}
            onChange={[Function]}
            onFocus={[Function]}
            placeholder="janesmith"
            type="text"
            value=""
          />
        </Input>
      </div>
    </FormGroup>
  </RenderInput>
  <RenderInput
    className="mb-4"
    handleBlur={[Function]}
    handleChange={[Function]}
    handleFocus={[Function]}
    id="password"
    label="Password"
    meta={
      Object {
        "error": null,
        "touched": false,
      }
    }
    name="password"
    placeholder="••••••••••"
    required={true}
    type="password"
    value=""
  >
    <FormGroup
      tag="div"
    >
      <div
        className="form-group"
      >
        <Label
          className="col-form-label"
          for="password"
          tag="label"
          widths={
            Array [
              "xs",
              "sm",
              "md",
              "lg",
              "xl",
            ]
          }
        >
          <label
            className="col-form-label  form-control-label"
            htmlFor="password"
          >
            Password
            <span
              className="text-danger"
            >
              *
            </span>
          </label>
        </Label>
        <Input
          className=""
          id="password"
          name="password"
          onBlur={[Function]}
          onChange={[Function]}
          onFocus={[Function]}
          placeholder="••••••••••"
          tag="p"
          type="password"
          value=""
        >
          <input
            className="form-control"
            id="password"
            name="password"
            onBlur={[Function]}
            onChange={[Function]}
            onFocus={[Function]}
            placeholder="••••••••••"
            type="password"
            value=""
          />
        </Input>
      </div>
    </FormGroup>
  </RenderInput>
  <RenderInput
    handleBlur={[Function]}
    handleChange={[Function]}
    handleFocus={[Function]}
    id="passwordconfirm"
    label="Password Confirmation"
    meta={
      Object {
        "error": null,
        "touched": false,
      }
    }
    name="passwordConfirm"
    required={true}
    type="password"
    value=""
  >
    <FormGroup
      tag="div"
    >
      <div
        className="form-group"
      >
        <Label
          className="col-form-label"
          for="passwordconfirm"
          tag="label"
          widths={
            Array [
              "xs",
              "sm",
              "md",
              "lg",
              "xl",
            ]
          }
        >
          <label
            className="col-form-label  form-control-label"
            htmlFor="passwordconfirm"
          >
            Password Confirmation
            <span
              className="text-danger"
            >
              *
            </span>
          </label>
        </Label>
        <Input
          className=""
          id="passwordconfirm"
          name="passwordConfirm"
          onBlur={[Function]}
          onChange={[Function]}
          onFocus={[Function]}
          tag="p"
          type="password"
          value=""
        >
          <input
            className="form-control"
            id="passwordconfirm"
            name="passwordConfirm"
            onBlur={[Function]}
            onChange={[Function]}
            onFocus={[Function]}
            type="password"
            value=""
          />
        </Input>
      </div>
    </FormGroup>
  </RenderInput>
  <RenderInput
    handleBlur={[Function]}
    handleChange={[Function]}
    handleFocus={[Function]}
    id="user-info"
    label="Tell Us Something About Yourself"
    meta={
      Object {
        "error": null,
        "touched": false,
      }
    }
    name="aboutMe"
    placeholder="I love to cook"
    required={false}
    type="textarea"
    value=""
  >
    <FormGroup
      tag="div"
    >
      <div
        className="form-group"
      >
        <Label
          className="col-form-label"
          for="user-info"
          tag="label"
          widths={
            Array [
              "xs",
              "sm",
              "md",
              "lg",
              "xl",
            ]
          }
        >
          <label
            className="col-form-label  form-control-label"
            htmlFor="user-info"
          >
            Tell Us Something About Yourself
          </label>
        </Label>
        <Input
          className=""
          id="user-info"
          name="aboutMe"
          onBlur={[Function]}
          onChange={[Function]}
          onFocus={[Function]}
          placeholder="I love to cook"
          tag="p"
          type="textarea"
          value=""
        >
          <textarea
            className="form-control"
            id="user-info"
            name="aboutMe"
            onBlur={[Function]}
            onChange={[Function]}
            onFocus={[Function]}
            placeholder="I love to cook"
            value=""
          />
        </Input>
      </div>
    </FormGroup>
  </RenderInput>
  <RenderInput
    handleBlur={[Function]}
    handleChange={[Function]}
    handleFocus={[Function]}
    id="occupation"
    label="Occupation"
    meta={
      Object {
        "error": null,
        "touched": false,
      }
    }
    name="occupation"
    placeholder="Chef"
    required={false}
    type="text"
    value=""
  >
    <FormGroup
      tag="div"
    >
      <div
        className="form-group"
      >
        <Label
          className="col-form-label"
          for="occupation"
          tag="label"
          widths={
            Array [
              "xs",
              "sm",
              "md",
              "lg",
              "xl",
            ]
          }
        >
          <label
            className="col-form-label  form-control-label"
            htmlFor="occupation"
          >
            Occupation
          </label>
        </Label>
        <Input
          className=""
          id="occupation"
          name="occupation"
          onBlur={[Function]}
          onChange={[Function]}
          onFocus={[Function]}
          placeholder="Chef"
          tag="p"
          type="text"
          value=""
        >
          <input
            className="form-control"
            id="occupation"
            name="occupation"
            onBlur={[Function]}
            onChange={[Function]}
            onFocus={[Function]}
            placeholder="Chef"
            type="text"
            value=""
          />
        </Input>
      </div>
    </FormGroup>
  </RenderInput>
</AuthForm>
`;
